react-scripts build failing - deploy on travis

> pancake-frontend@0.1.0 build /home/travis/build/ServiceInnovationLab/pancake-frontend
> react-scripts build
Creating an optimized production build...
Treating warnings as errors because process.env.CI = true.
Most CI servers set it automatically.
Failed to compile.
./src/index.js
  Line 5:  'applyMiddleware' is defined but never used  no-unused-vars
  Line 7:  'thunk' is defined but never used            no-unused-vars
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! pancake-frontend@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pancake-frontend@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-04-15T23_00_01_195Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! pancake-frontend@0.1.0 predeploy: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pancake-frontend@0.1.0 predeploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-04-15T23_00_01_208Z-debug.log
Script failed with status 1
failed to deploy

--- a/src/index.js
+++ b/src/index.js
@@ -15,8 +15,8 @@ import Footer from './components/Footer';
 const store = createStore(combineReducers({
   reducers,
   form: formReducer
-}), window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
-// }), applyMiddleware(thunk));
+// }), window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
+}), applyMiddleware(thunk));
 
 class App extends React.Component {
   render(){

